shorty/autoconfig.py

```diff
@@ -10,4 +10,8 @@
         'nuit',
     ],
     'NUIT_GLOBAL_TITLE': 'Shorty',
+    'CSP_SCRIPT_SRC': (
+        "'self'",
+        "https://cdnjs.cloudflare.com",
```

Comment:
Do we trust that *any* library on the cdnjs site is trustworthy? We probably want to whitelist the specific libraries we're interested in